hudi-utilities/src/test/resources/delta-streamer-config/target.avsc

```diff
@@ -70,7 +70,7 @@
     "name" : "height",
     "type" : {
       "type" : "fixed",
-      "name" : "abc",
```

Comment:
also, can you help me understand why we change the name here to fixed. name could be anything in general right.

Reply:
Because when we convert Spark schema to avro schema [here](https://github.com/apache/spark/blob/master/external/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala#L177), it always name it as `fixed`. If we name it different, then this avro schema and avro schema created from spark schema would not match